app/experimenter/nimbus-ui/src/components/PageEditAudience/FormAudience/index.test.tsx

```diff
@@ -50,7 +50,7 @@ describe("FormAudience", () => {
           targetingConfigs: [
             {
               label: "No Targeting",
-              value: "NO_TARGETING",
+              value: "",
```

Comment:
Yes this is the correct value for the `No Targeting` option, as defined here:

https://github.com/mozilla/experimenter/blob/main/app/experimenter/experiments/constants/nimbus.py#L160

and returned through the getConfig graphql call, so good catch 👍